lib/common/output_text.c

```diff
@@ -34,6 +34,7 @@ free_list_data(gpointer data) {
 
     free(list_data->singular_noun);
     free(list_data->plural_noun);
+    free(list_data);
```

Comment:
This same problem will affect `tools/crm_mon_curses.c`.  Might as well add that to this patch and break it out into a separate PR so we can merge it while talking about the rest of this PR.

Reply:
#3814